Subject: cosmetic changes (newlines, space) to some scripts
Author: Robin Gareus <robin@gareus.org>
Last-Update: 2010-02-14
Index: campcaster/etc/Makefile.in
===================================================================
--- campcaster.orig/etc/Makefile.in	2010-02-15 00:33:59.000000000 +0100
+++ campcaster/etc/Makefile.in	2010-02-15 00:34:03.000000000 +0100
@@ -66,7 +66,7 @@
 WWW_DOCROOT             = @WWW_DOCROOT@
 WWW_PORT                = @WWW_PORT@
 SCHEDULER_PORT          = @SCHEDULER_PORT@
-DB_SERVER         		= @DB_SERVER@
+DB_SERVER         	= @DB_SERVER@
 DATABASE                = @DATABASE@
 DB_USER                 = @DB_USER@
 DB_PASSWORD             = @DB_PASSWORD@
Index: campcaster/src/modules/storageServer/bin/createDatabase.sh
===================================================================
--- campcaster.orig/src/modules/storageServer/bin/createDatabase.sh	2010-02-15 00:34:12.000000000 +0100
+++ campcaster/src/modules/storageServer/bin/createDatabase.sh	2010-02-15 00:35:04.000000000 +0100
@@ -24,7 +24,8 @@
 #   Author   : $Author$
 #   Version  : $Revision$
 #   Location : $URL$
-#-------------------------------------------------------------------------------                                                                                
+#-------------------------------------------------------------------------------
+
 #-------------------------------------------------------------------------------
 #  This script reates the database used by Campcaster
 #
Index: campcaster/src/products/scheduler/bin/campcaster-scheduler.sh
===================================================================
--- campcaster.orig/src/products/scheduler/bin/campcaster-scheduler.sh	2010-02-15 00:35:07.000000000 +0100
+++ campcaster/src/products/scheduler/bin/campcaster-scheduler.sh	2010-02-15 00:35:24.000000000 +0100
@@ -1,5 +1,6 @@
 #!/bin/bash
-#-------------------------------------------------------------------------------#   Copyright (c) 2004 Media Development Loan Fund
+#-------------------------------------------------------------------------------
+#   Copyright (c) 2004 Media Development Loan Fund
 #
 #   This file is part of the Campcaster project.
 #   http://campcaster.campware.org/
