Index: campcaster/src/modules/playlistExecutor/src/GstreamerPlayContext.h
===================================================================
--- campcaster.orig/src/modules/playlistExecutor/src/GstreamerPlayContext.h	2010-02-26 13:56:18.000000000 +0100
+++ campcaster/src/modules/playlistExecutor/src/GstreamerPlayContext.h	2010-02-26 13:58:37.000000000 +0100
@@ -300,6 +300,7 @@
         // and ALSA can emulate it.
         // const bool oss = m_audioDevice.find("/dev") == 0;
         const bool autosink = m_audioDevice.find("auto") == 0;
+        const bool gconfsink = m_audioDevice.find("gconf") == 0;
 
         m_sink = gst_bin_new ("audiobin");
         if(m_sink == NULL){
@@ -309,7 +310,9 @@
         audiopad = gst_element_get_pad (conv, "sink");
 
         // set the string to be sent to gstreamer. the option here is to set it to autoaudiosink.
-        GstElement *sink = (autosink ? gst_element_factory_make("autoaudiosink", NULL) : gst_element_factory_make("alsasink", NULL));
+        GstElement *sink = (autosink ? gst_element_factory_make("autoaudiosink", NULL) :
+			   (gconfsink? gst_element_factory_make("gconfaudiosink", NULL):
+				       gst_element_factory_make("alsasink", NULL)));
         if(sink == NULL){
             return false;
         }
